Sources/FCM/Helpers/FCM+SendMessage.swift

```diff
@@ -10,6 +10,8 @@ extension FCM {
         guard let configuration = self.configuration else {
             fatalError("FCM not configured. Use app.fcm.configuration = ...")
         }
+        var message = message
```

Comment:
Im assuming nothing else is relying on properties being set on the original reference? Unlikely but worth checking

Reply:
It's a struct so shouldn't be a problem